eng/scripts/update-npm-dependencies.ps1

```diff
@@ -1,132 +0,0 @@
-
-    if ($null -eq $commonOptionalDependencyVersion) {
-        $commonOptionalDependencyVersion = $optionalDependencyVersion
-    }
-
-    Write-Host "Adding $optionalDependencyName@$optionalDependencyVersion to the cache"
-    if (-not $WhatIf) {
-        npm cache add $optionalDependencyName@$optionalDependencyVersion
-    }
-}
-
-if ($null -ne $commonOptionalDependencyVersion) {
-    Write-Host "Adding @rollup/wasm-node@$commonOptionalDependencyVersion to the cache"
-    if (-not $WhatIf) {
-        npm cache add "@rollup/wasm-node@$commonOptionalDependencyVersion"
-    }
-}
-
-Write-Host "Verifying the cache"
-if(-not $WhatIf) {
-    npm cache verify
-}
-
-# Navigate to the Node-Externals submodule
-# Checkout a branch named infrastructure/update-npm-package-cache-<date>
-# Stage the changes in the folder
-# Commit the changes with the message "Updated npm package cache <date>"
-# Use the GH CLI to create a PR for the branch in the origin remote
-
-Push-Location src/submodules/Node-Externals
-$branchName = "infrastructure/update-npm-package-cache-$(Get-Date -Format 'yyyy-MM-dd')"
-if (-not $WhatIf) {
-    git branch -D $branchName 2>$null
-    git checkout -b $branchName
-    git add .
-    git commit -m "Updated npm package cache $(Get-Date -Format 'yyyy-MM-dd')"
-}
-
-if ($WhatIf -or $SkipPullRequestCreation) {
-    Write-Host "Skipping pull request creation for Node-Externals submodule"
-}
-else {
-    Write-Host "Creating pull request for Node-Externals submodule"
-    git branch --set-upstream-to=origin/main
-    git push origin $branchName`:$branchName --force;
-    gh repo set-default dotnet/Node-Externals
-    gh pr create --base main --head "infrastructure/update-npm-package-cache-$(Get-Date -Format 'yyyy-MM-dd')" --title "[Infrastructure] Updated npm package cache $(Get-Date -Format 'yyyy-MM-dd')" --body "Do not merge this PR until the one in aspnetcore passes all checks."
-}
-
-## Navigate to the root of the repository
-## Checkout a branch named infrastructure/update-npm-package-cache-<date>
-## Stage the changes in the folder
-## Commit the changes with the message "Updated npm package cache <date>"
-## Use the GH CLI to create a PR for the branch in the origin remote
-Pop-Location
-if(-not $WhatIf) {
-    git branch -D $branchName 2>$null
-    git checkout -b $branchName
-    git add .
-    git commit -m "Updated npm package cache $(Get-Date -Format 'yyyy-MM-dd')"
-}
-
-if ($WhatIf -or $SkipPullRequestCreation) {
-    Write-Host "Skipping pull request creation for the root of the repository"
-}
-else {
-    Write-Host "Creating pull request for the root of the repository"
-    git branch --set-upstream-to=origin/main
-    git push origin $branchName`:$branchName --force;
-    gh repo set-default dotnet/aspnetcore
-    gh pr create --base main --head $branchName --title "[Infrastructure] Updated npm package cache $(Get-Date -Format 'yyyy-MM-dd')" --body ""
-}
```

Comment:
The script is still useful, just not the submodules part.  Leave it around

Reply:
Done